Altis_Life.Altis/core/vehicle/fn_openInventory.sqf

```diff
@@ -30,6 +30,13 @@ ctrlSetText[3504,format [(localize "STR_MISC_Weight")+ " %1/%2",_veh_data select
 [_vehicle] call life_fnc_vehInventory;
 life_trunk_vehicle = _vehicle;
 
+_vehicle spawn {
+    waitUntil {
+        if ((_this getVariable ["trunk_in_use_by",player]) != player) then {closeDialog 0; hint localize "STR_MISC_VehInvUse"};
```

Comment:
Thanks for doing this, you do need to actually make the string in the stringtable though! If you need any help with that feel free to join the discord.